File: Aulas/banco/src/Pessoa.java
public class Pessoa
{
    private String cpf;     // cpf imutável, não terá setter
    private String nome;    
    
    /* Construtor que obriga a definir cpf e nome na criação */
    public Pessoa(String cpf, String nome) {
        this.cpf = cpf;
        this.nome = nome;
    }
    public String getCpf() {
        return cpf;
    }
    public String getNome() {
        return nome;
    }
    public void setNome(String nome) {
        this.nome = nome;
    }
    
}

File: Aulas/C++/Aula 2/pessoa.cpp
#include "pessoa.hpp"

#include <iostream>

Pessoa::Pessoa(const string& nome, int idade) : nome(nome), idade(idade) 
{
    std::cout << "Construímos uma pessoa de nome " << nome << " e idade " << idade << std::endl;
};

void Pessoa::setNome(const string& nome)
{
    this->nome=nome;
}

string Pessoa::getNome() const
{
    return nome;
}
